core/templates/domain/admin/admin-backend-api.service.spec.ts

```diff
@@ -197,4 +205,124 @@ describe('Admin backend api service', () => {
+
+      let req = httpTestingController.expectOne('/adminhandler');
+      expect(req.request.method).toEqual('POST');
+      expect(req.request.body).toEqual(payload);
+      req.flush(200);
+      flushMicrotasks();
+
+      expect(successHandler).toHaveBeenCalled();
+      expect(failHandler).not.toHaveBeenCalled();
+    }
+    ));
+
+  it('should request to cancel the job given its id' +
+    'and type when calling cancelJobAsync', fakeAsync(() => {
+    let jobId = 'AuditContributorsOneOffJob-1608291840709-843';
+    let jobType = 'AuditContributorsOneOffJob';
+    let payload = {
+      action: 'cancel_job',
+      job_id: jobId,
+      job_type: jobType
+    };
+    abas.cancelJobAsync(jobId, jobType).then(successHandler, failHandler);
+
+    let req = httpTestingController.expectOne('/adminhandler');
+    expect(req.request.method).toEqual('POST');
+    expect(req.request.body).toEqual(payload);
+    req.flush(200);
+    flushMicrotasks();
+
+    expect(successHandler).toHaveBeenCalled();
+    expect(failHandler).not.toHaveBeenCalled();
+  }
+  ));
+
+  it('should request to start computation given the job' +
+    'name when calling startComputationAsync', fakeAsync(() => {
+    let computationType = 'FeedbackAnalyticsAggregator';
+    let payload = {
+      action: 'start_computation',
+      computation_type: computationType
+    };
+    abas.startComputationAsync(computationType)
+      .then(successHandler, failHandler);
+
+    let req = httpTestingController.expectOne('/adminhandler');
+    expect(req.request.method).toEqual('POST');
+    expect(req.request.body).toEqual(payload);
+    req.flush(200);
+    flushMicrotasks();
+
+    expect(successHandler).toHaveBeenCalled();
+    expect(failHandler).not.toHaveBeenCalled();
+  }
+  ));
+
+  it('should request to stop computation given the job' +
+    'name when calling stopComputationAsync', fakeAsync(() => {
+    let computationType = 'FeedbackAnalyticsAggregator';
+    let payload = {
+      action: 'stop_computation',
+      computation_type: computationType
+    };
+    abas.stopComputationAsync(computationType)
+      .then(successHandler, failHandler);
+
+    let req = httpTestingController.expectOne('/adminhandler');
+    expect(req.request.method).toEqual('POST');
+    expect(req.request.body).toEqual(payload);
+    req.flush(200);
+    flushMicrotasks();
+
+    expect(successHandler).toHaveBeenCalled();
+    expect(failHandler).not.toHaveBeenCalled();
+  }
+  ));
+
+  it('should request to show the output of valid' +
+    'jobs when calling showJobOutputAsync', fakeAsync(() => {
+    let jobId = 'FeedbackAnalyticsMRJobManager-1608932124519-748';
+    let adminJobOutputUrl = '/adminjoboutput?job_id=FeedbackAnalytics' +
+      'MRJobManager-1608932124519-748';
+    abas.showJobOutputAsync(jobId).then(successHandler, failHandler);
+
+    let req = httpTestingController.expectOne(adminJobOutputUrl);
+    expect(req.request.method).toEqual('GET');
```

Comment:
assert the response here. (Ditto in other places where we are expecting to receive a response.)

Reply:
done.